### PR TITLE
Merge master to preview

### PR DESCRIPTION
**Pull request by vinagesh**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-java/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.
- I submitted this PR against the correct branch: 
  - [ ] This pull-request is submitted against the `master` branch. 

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: vsts/generate_javadocs_branch.ps1 [hunk 1]
```diff
@@ -56,85 +63,101 @@ function CreateJavadocReleaseBranch(
     TestLastExitCode  # stop if we can't create the branch
 
     Write-Host "Copying generated javadocs to replace current javadocs"
-    Set-Location apidocs
+    Set-Location apidocs
```
Comment (drwill-ms) [inline vsts/generate_javadocs_branch.ps1:66]: extra spaces appended

File: vsts/generate_javadocs_branch.ps1 [hunk 2]
```diff
@@ -56,85 +63,101 @@ function CreateJavadocReleaseBranch(
...
+    if ($UpdateTpmProviderDocs -eq "True")
+    {
+        Remove-Item ..\$FolderName\provisioning\security\tpm-provider -Force -Recurse
+        New-Item -Path ..\$FolderName\provisioning\security\tpm-provider -ItemType Directory
+        Copy-Item -Force -Path .\provisioning\security\tpm-provider\* -Destination ..\$FolderName\provisioning\security\tpm-provider
+        Copy-Item -Recurse -Force -Path .\provisioning\security\tpm-provider\com -Destination ..\$FolderName\provisioning\security\tpm-provider
+    }
 
-    if ($UpdateDepsDocs -eq "True")
+    if ($UpdateDiceProviderEmulatorDocs -eq "True")
```
Comment (drwill-ms) [inline vsts/generate_javadocs_branch.ps1:124]: Dice?

**Review by drwill-ms — APPROVED** [review]

File: vsts/generate_javadocs_branch.ps1 (hunk 2 above)
Comment (vinagesh) [reply vsts/generate_javadocs_branch.ps1:124]: It is dice. I'm not sure what that package is though. We should ask Tim.

File: vsts/generate_javadocs_branch.ps1 (hunk 1 above)
Comment (vinagesh) [reply vsts/generate_javadocs_branch.ps1:66]: I'll have to make this fix in master. This is just a merge commit. Let me send out a PR and then merge again.

Comment (vinagesh) [reply vsts/generate_javadocs_branch.ps1:66]: Here is the PR - https://github.com/Azure/azure-iot-sdk-java/pull/1031

**Pull request merged**